ONBOARDING.md

```diff
@@ -0,0 +1,9 @@
+## TSC Onboarding Document
+
+1. Add the user to the slack @tsc team. This is done in https://adoptopenjdk.slack.com/admin/user_groups but requires an owner/admin of slack to do this.
+
+1. Add the user to the GitHub @tsc team: https://github.com/orgs/AdoptOpenJDK/teams/tsc/members
+
+1. Add the user to the Public Members page in the TSC repository: https://github.com/AdoptOpenJDK/TSC#current-members
+
+1. Create an `@adoptopenjdk.net` email for them. George Adams is the administrator and can do this.
```

Comment:
This requires any GSuite admin, right?

Reply:
Ahh yes you're right, I think I made you and Martijn admins too right?

Reply:
Not sure if there is a handy e-mail alias for admins.  I'm just trying to avoid a single person point of 'failure' in the process.